articles/automatic-software-install-in-fleet.md

```diff
@@ -61,7 +61,7 @@ Upon failure of the selected policy, the selected software installation will be
 
 ## Templates for policy queries
 
-Following are some templates for the policy SQL queries for each package type.
+Use the following policy templates to see if the software is already installed. Fleet uses these templates to automatically install software.
```

Comment:
```
Fleet uses these templates to automatically install software.
```

@mostlikelee can you please sanity check us here? Are these the policy templates Fleet uses to auto generate policies?